src/api/app/views/webui2/webui/staging_workflows/ignored_requests/_create_ignored_request_dialog.html.haml

```diff
@@ -0,0 +1,17 @@
+.modal.fade#ignore-request-modal{ tabindex: -1, role: 'dialog', aria: { labelledby: 'ignore_request-modal-label', hidden: true } }
```

Comment:
I really don't like this workflow :thinking: As a user I have to remember the number of the request (and there is not even e.g. a autocomplete). I would expect on the request page to have a button to ignore this request.

Reply:
which brings us to the same problem we see in #6202: there is no page for incoming staging requests - this one would be perfect to gain an ignore button. So I would wait with such details

Reply:
+1 for listing staged requests (and requests with open reviews) and have an ignore button for each of them

Reply:
@ChrisBr:

> I really don't like this workflow thinking As a user I have to remember the number of the request (and there is not even e.g. a autocomplete). I would expect on the request page to have a button to ignore this request.

Can we leave that to another pull request, or do you think it is really needed in this page? I will leave like it is, for now.

Reply:
@bgeuken:

> +1 for listing staged requests (and requests with open reviews) and have an ignore button for each of them

Is is for you a requirement, really needed in this page? I vote for leaving like it is, for now.

Reply:
I won't block this PR of this but honestly I would prefer to do the things right. It just doesn't make much sense to enter a number and it will bring us in trouble for sure....

Reply:
I think it can be done in another PR, but I agree it needs to be done.

Reply:
@eduardoj No, it's not. But I like the idea.

Sorry for the late reply.